Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         }
         
         eventDetailViewController.theEvent = event;
-        //eventDetailViewController.title = event.title;
```

Comment:
Yep! Make sure you remove the commented code. 

> On Apr 17, 2016, at 2:43 PM, AFrench <notifications@github.com> wrote:
> 
> In Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m:
> 
> > @@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
> >          }
> >          
> >          eventDetailViewController.theEvent = event;
> > -        //eventDetailViewController.title = event.title;
> 
> Good?
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
>